src/app/stack/report-summary/report-summary-card/report-summary-title/report-summary-title.component.html

```diff
@@ -2,8 +2,14 @@
     <div class="card-pf-title">
         <span [class]="title?.titleIcon" aria-hidden="true"></span>
         {{title?.titleText}}
+        <span *ngIf="title?.titleText === 'Security Issues' || title?.titleText ===  'Dependency Details'">
+            <span class="branding">Powered By
+                <img src="../../../../../assets/icon/snyk-logo-monochrome.png" alt="snyk-mono" class="monologo" />
```

Comment:
if assets directory is in root, you can simply refer as `/assets/icon/snyk-logo-monochrome.png`
```suggestion
                <img src="/assets/icon/snyk-logo-monochrome.png" alt="snyk-mono" class="monologo" />
```

Reply:
Fixed